docs/src/main/sphinx/release/release-352.md

```diff
@@ -59,8 +59,9 @@
 * Match columns by name rather than by index by default for Parquet files. ({issue}`6479`)
 * Remove the `hive.partition-use-column-names` configuration property and the
   `partition_use_column_names ` session property. This is now determined automatically. ({issue}`6479`)
-* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+* Support reading timestamps with microsecond or nanosecond precision (as configured with
```

Comment:
I don't have a string opinion whether plural or singular, but let's keep the original wording.